src/compiler/test-file/formats/dev-tools/commands/factory.ts

```diff
@@ -1,21 +1,33 @@
 import { DEVTOOLS_COMMAND_TYPE, DevToolsRecorderStep } from '../types';
 import { NavigateCommandTransformer } from './navigate';
 import { SetViewportCommandTransformer } from './set-viewport';
-import { ClickCommandTransformer } from './click';
 import { ChangeCommandTransformer } from './change';
 import { KeyDownCommandTransformer } from './key-down';
 import { KeyUpCommandTransformer } from './key-up';
 import { ScrollCommandTransformer } from './scroll';
 import { WaitForExpressionCommandTransformer } from './wait-for-expression';
 import { WaitForElementCommandTransformer } from './wait-for-element';
 import { CommandTransformerBase } from './base';
+import { HoverCommandTransformer } from "./hover";
+
+import {
+    ClickCommandTransformer,
+    DoubleClickCommandTransformer,
+    RightClickCommandTransformer,
+} from './click';
 
 export class CommandTransformerFactory {
     static create (step: DevToolsRecorderStep, callsite: number): CommandTransformerBase | null {
         switch (step.type) {
             case DEVTOOLS_COMMAND_TYPE.navigate: return new NavigateCommandTransformer(step, callsite);
             case DEVTOOLS_COMMAND_TYPE.setViewport: return new SetViewportCommandTransformer(step, callsite);
-            case DEVTOOLS_COMMAND_TYPE.click: return new ClickCommandTransformer(step, callsite);
+            case DEVTOOLS_COMMAND_TYPE.click: {
+                if (step.button === 'secondary')
+                    return new RightClickCommandTransformer(step, callsite);
+                return new ClickCommandTransformer(step, callsite);
+            }
+            case DEVTOOLS_COMMAND_TYPE.dblClick: return new DoubleClickCommandTransformer(step, callsite);
```

Comment:
We should fail test run each time once we face with unfamiliar command name.
![image](https://user-images.githubusercontent.com/12034551/188841760-efb43de5-3796-46b2-bc6e-8ec54d20a801.png)
Otherwise we will have to update our DevToolsCompiler for each DevTools recorder update. Let's just add warning and skip this command

Reply:
Proposal 1 (If we ignore the action):

```md
WARNING: TestCafe encountered an unsupported action ("$ACTION") in a Chrome User Flow file ("$FLOW_PATH"). The action will be skipped during test execution, which can alter the course of the test. Remove the action to disable this warning. Refer to the following article for the definitive list of supported Chrome User Flow actions: https://testcafe.io/documentation/403998/guides/experimental-capabilities/chrome-replay-support#supported-replay-actions
```

Proposal 2 (if we do decide to terminate the test run):
```md
ERROR: TestCafe terminated the test run because the "$FLOW_PATH" file contains an unknown Chrome User Flow action ("$ACTION").  Remove the action to continue. Refer to the following article for the definitive list of supported. Chrome User Flow actions: https://testcafe.io/documentation/403998/guides/experimental-capabilities/chrome-replay-support#supported-replay-actions
```

Reply:
We agreed to terminate the test run in case of unknown action